### ISSUE TITLE
Error saving plan after agent name change??

### ISSUE THREAD

**Issue by fosterlynn** [Carbon-Farm-Network/app-carbon-farm-network#86]
Below is what it looks like (an assortment).  I've tried different conditions.  My other instance from today does not have this problem (has other ones).  From the error message and the fact that I changed the "Carbon Farm Network" name to "Carbon Farm Net", I'm wondering if there is some hard-coded check reference that?  Seems unlikely, I'm just guessing.  I have to reboot my computer to try it between tests, otherwise I just get that screen forever, even if I close CFN and launcher.

![Screenshot from 2024-02-13 17-59-50](https://github.com/Carbon-Farm-Network/app-carbon-farm-network/assets/3776081/0749e000-9d07-4942-a737-dc4b7e1b384f)
![Screenshot from 2024-02-13 16-52-01](https://github.com/Carbon-Farm-Network/app-carbon-farm-network/assets/3776081/f7350256-8566-4cea-8b23-8c54789fdfa4)
![Screenshot from 2024-02-13 15-15-24](https://github.com/Carbon-Farm-Network/app-carbon-farm-network/assets/3776081/fac30e56-9e6b-4dcb-a940-c6c4b34f3af8)
![Screenshot from 2024-02-13 14-40-38](https://github.com/Carbon-Farm-Network/app-carbon-farm-network/assets/3776081/537e5bac-ec73-4656-b12b-8a617c10112e)

**Comment by LeosPrograms** [comment]
Yep, it's hard-coded. Should be changed to look for "network" role.

**Comment by fosterlynn** [comment]
That was it, I changed the name back, and it saved and returned.  Leaving the issue open so we can change it to check for Network.  That is still a hack, but when we actually make the roles user maintainable, we can fix that to be a flag or something.
